src/main/java/org/jenkinsci/plugins/docker/workflow/client/DockerClient.java

```diff
@@ -156,8 +155,28 @@ public void rm(@Nonnull EnvVars launchEnv, @Nonnull String containerId) throws I
             return null;
         }
     }
-
-    private Date getCreatedDate(@Nonnull EnvVars launchEnv, @Nonnull String objectId) throws IOException, InterruptedException {
+    
+    /**
+     * Inspect a docker image/container.
+     * @param launchEnv Docker client launch environment.
+     * @param objectId The image/container ID.
+     * @param fieldPath The data path of the data required e.g. {@code .NetworkSettings.IPAddress}.
+     * @return The inspected field value
+     * @throws IOException Execution error. Also fails if cannot retrieve the requested field from the request
+     * @since TODO
+     */
+    public @Nonnull String inspectRequiredField(@Nonnull EnvVars launchEnv, @Nonnull String objectId, 
+            @Nonnull String fieldPath) throws IOException, InterruptedException {
+        final LaunchResult result = launch(launchEnv, true, "inspect", "-f", 
+                String.format("{{%s}}", fieldPath), objectId);
+        String fieldValue = result.getStatus() == 0 ? hudson.Util.fixEmpty(result.getOut()) : null;
+        if (fieldValue == null) {
```

Comment:
I would expect this to just be defined in terms of `inspect`, to avoid duplicating the details of launching the command.